src/kernel_gauss_histogram.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+#include "kernels.h"
+
+//blur histogram using kernel of ones
```

Comment:
"of ones" ?

Reply:
i meant kernel where all the fields consisted of number one (but it is divided by the sum of the kernel so the intensity doesn't change)

Reply:
maybe better commentar would be more suitable :D